checker/jdk/Makefile.jdk

```diff
@@ -30,8 +30,8 @@ build: classes
 
 ############## Validate annotated jdk against missing methods ##############
 
-SOURCES = $(shell find src/java src/javax -name '*.java')
-# CLASSES = $(shell find src/java src/javax -name '*.class')
+SOURCES = $(shell find src/java src/javax src/org/w3c src/org/xml -name '*.java')
```

Comment:
The JDK, including langtools (= javac, javadoc, etc), will also be annotated in an external repo soon.